backend/grants/models.py

```diff
@@ -25,6 +25,16 @@
 
 
 class Grant(TimeStampedModel):
+    class AgeGroup(models.TextChoices):
+        RANGE_LESS_THAN_10 = "range_less_than_10", _("10 years old or under")
```

Comment:
I am not 100% convinced about having this "less than 10 years old" option, it is not like someone is going to use it 🤔 

I need to think about it a bit